src/legacy/core_plugins/console/np_ready/public/application/containers/editor/legacy/use_ui_ace_keyboard_mode.tsx

```diff
@@ -0,0 +1,109 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React, { useEffect, useRef } from 'react';
+import { render, unmountComponentAtNode } from 'react-dom';
+import { keyCodes } from '@elastic/eui';
+
+const Overlay = ({
+  onMount,
+  onKeyDown,
+}: {
+  onMount: (ref: HTMLDivElement) => void;
+  onKeyDown: (ev: React.KeyboardEvent) => void;
+}) => (
+  // The point of this element is for accessibility purposes, so ignore eslint error
+  // in this case
+  //
+  // eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions
+  <div
+    ref={onMount}
+    onKeyDown={onKeyDown}
+    className="kbnUiAceKeyboardHint"
+    tabIndex={0}
+    role="application"
+  >
+    <p className="kuiText kuiVerticalRhythmSmall">Press Enter to start editing.</p>
```

Comment:
We can convert this to EUI as well. I'd use https://github.com/elastic/eui/blob/master/src/components/code_editor/code_editor.js#L165 as a reference, but maybe we can use EuiText in place of `<p className="euiText">`.